src/app/views/common/monaco/Monaco.tsx

```diff
@@ -1,14 +1,14 @@
 import { FocusZone } from '@fluentui/react';
 import React from 'react';
-import MonacoEditor, { ChangeHandler } from 'react-monaco-editor';
```

Comment:
Does this package`react-monaco-editor` need to be uninstalled since it's no longer being used.

Reply:
Yeah, let me uninstall it